pyvo/make_vid.py

```diff
@@ -183,7 +183,7 @@ def make_pyvo(
     if praha:
         overlay = export_template.exported_slide('slide-overlay', duration=main.duration)
         main |= overlay
-    main = main.faded_out(0.5)
+    main = main.faded_out(0.5).faded_in(0.5)
```

Comment:
I think the fade-in should be skipped for `speaker_only` videos

Reply:
I did some speaker-only videos and I don't think it's bad for them either. Anyway, feel free to ignore these commits until you have the time to see the results.